dev/VSIX/Extension/Cs/Common/VSPackage.resx

```diff
@@ -151,60 +151,60 @@
     <value>[Experimental] Resources File (.resw)</value>
   </data>
   <data name="1015" xml:space="preserve">
-    <value>A file for storing string and conditional resources for apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A file for storing string and conditional resources for WinUI 3 apps.</value>
   </data>
   <data name="1017" xml:space="preserve">
-    <value>Custom Control (WinUI 3)</value>
+    <value>Templated Control</value>
   </data>
   <data name="1018" xml:space="preserve">
-    <value>[Experimental] Custom Control (WinUI 3)</value>
+    <value>[Experimental] Templated Control</value>
   </data>
   <data name="1019" xml:space="preserve">
-    <value>A custom control for apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A custom control for WinUI 3 apps.</value>
   </data>
   <data name="1021" xml:space="preserve">
-    <value>User Control (WinUI 3)</value>
+    <value>User Control</value>
   </data>
   <data name="1022" xml:space="preserve">
-    <value>[Experimental] User Control (WinUI 3)</value>
+    <value>[Experimental] User Control</value>
   </data>
   <data name="1023" xml:space="preserve">
-    <value>A user control for apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A user control for WinUI 3 apps.</value>
   </data>
   <data name="1025" xml:space="preserve">
-    <value>Class Library (WinUI 3 in Desktop)</value>
+    <value>Class Library</value>
   </data>
   <data name="1026" xml:space="preserve">
-    <value>[Experimental] Class Library (WinUI 3 in Desktop)</value>
+    <value>[Experimental] Class Library</value>
   </data>
   <data name="1027" xml:space="preserve">
-    <value>A project for creating a managed class library (.dll) for Desktop apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A project for creating a managed class library (.dll) for WinUI 3 apps.</value>
   </data>
   <data name="1029" xml:space="preserve">
-    <value>Blank App, Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)</value>
+    <value>WinUI Blank App (Packaged with Windows Application Packaging Project)</value>
```

Comment:
What WAP stands for is described in its description.
```suggestion
    <value>WinUI Blank App (Packaged with WAP)</value>
```

Reply:
Why not just
```suggestion
    <value>WinUI App (Packaged)</value>
```

Reply:
This is for single-packing project template, already exists.

Reply:
Oops forgot it was the same

```suggestion
    <value>WinUI App (Packaged with Packaging Project)</value>
```

This sounds better imo

Reply:
Well, no.

Reply:
Agree that keeping Blank in the title makes sense; for alignment with other frameworks and for any potential other templates that might ship as part of WASDK.

@0x5bfa do you have examples where WAP is used vs. "Windows Application Packaging Project"?

Reply:
It's really long though. 
Developers will understand the meaning of "WinUI App" and it's a simple name that represents WinUI.
Imagine if Class Library was called "Class Library as Runtime Assembly (Packaged as Link Library)" that would be too long.
I think it should be good to be simple

Reply:
That was misleading, WAP isn't used across the platform (only Empty/Blank). But the folks have kept using that afaik.

For example: 
- https://developercommunity.visualstudio.com/t/windows-application-packaging-project-wap-causes-e/1347957
- Previous template
![IMG_0705](https://github.com/user-attachments/assets/5254547f-7b50-4379-b9ec-23eb3edfed9e)

Reply:
FYI, `WinUI Application` is consistent?

![IMG_0706](https://github.com/user-attachments/assets/2103c6e4-b3ed-4e75-8ec6-03f29c94b7c1)

Reply:
Ah, I see. Yeah things seem to be a bit different across WPF and UWP.

Does it make sense to keep things as close as possible to UWP?
![image](https://github.com/user-attachments/assets/b13cd38c-c186-4a32-801c-e9fb68ecf7d0)

Reply:
Yeah